src/main/java/com/code_intelligence/jazzer/mutation/mutator/proto/BuilderMutatorFactory.java

```diff
@@ -243,22 +230,81 @@ public Builder detach(Builder builder) {
    * type with a message field B and B in turn has a message field of type A, then the mutators for
    * A and B will reference each other, forming a cycle.
    */
-  private final HashMap<Class<? extends Builder>, SerializingMutator<Builder>> internedMutators =
+  private final HashMap<CacheKey, SerializingMutator<? extends Builder>> internedMutators =
       new HashMap<>();
 
+  private SerializingMutator<Any.Builder> createMutatorForAny(
```

Comment:
Align with the naming schema of the other methods, so `mutatorForAnyField`?

Reply:
Changed it to `mutatorForAny` as it creates a mutator for the entire builder, not a field.